loops.js

```diff
@@ -44,7 +49,17 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i);
+    }
+    if (i >= 3 && i < 6) {
+        console.log('> ' + i);
+    }
+    if (i >= 6 && i < 10) {
```

Comment:
```suggestion
    } else if (i < 6) {
        console.log('> ' + i);
    } else if (i < 10) {
```
Omdat de eerste if false heeft opgeleverd, moet het bij de tweede if het getal wel groter of gelijk zijn aan 3